src/site/content/en/vitals/index.md

```diff
@@ -0,0 +1,308 @@
+[usage](https://github.com/GoogleChrome/web-vitals#usage) and
+[API](https://github.com/GoogleChrome/web-vitals#api) details):
+
+```js
+import {getCLS, getFID, getLCP} from 'web-vitals';
+
+function sendToAnalytics(metric) {
+  const body = JSON.stringify(metric);
+  // Use `navigator.sendBeacon()` if available, falling back to `fetch()`.
+  (navigator.sendBeacon && navigator.sendBeacon('/analytics', body)) ||
+      fetch('/analytics', {body, method: 'POST', keepalive: true});
+}
+
+getCLS(sendToAnalytics);
+getFID(sendToAnalytics);
+getLCP(sendToAnalytics);
+```
+
+You can also report on each of the Core Web Vitals without writing any code
+using the [Web Vitals Chrome
+Extension](https://github.com/GoogleChrome/web-vitals-extension). This extension
+uses the [web-vitals](https://github.com/GoogleChrome/web-vitals) library to
+measure each of these metrics and display them to users as they browse the web.
+
+This extension can be helpful in understanding the performance of your own
+sites, your competitor's sites, and the web at large.
+
+<table class="w-table-wrapper">
+  <thead>
+    <tr>
+      <th>&nbsp;</th>
+      <th>LCP</th>
+      <th>FID</th>
+      <th>CLS</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/GoogleChrome/web-vitals">web-vitals</a></td>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/GoogleChrome/web-vitals-extension">
+        Web Vitals Extension</a></td>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+  </tbody>
+</table>
+
+Alternatively, developers who prefer to measure these metrics directly via the
+underlying web APIs can refer to these metric guides for implementation details:
+
+- [Measure LCP in JavaScript](https://web.dev/lcp/#measure-lcp-in-javascript)
+- [Measure FID in JavaScript](https://web.dev/fid/#measure-fid-in-javascript)
+- [Measure CLS in JavaScript](https://web.dev/cls/#measure-cls-in-javascript)
+
+#### Lab tools to measure Core Web Vitals
+
+While all of the Core Web Vitals are, first and foremost, field metrics, many of
+them are also measurable in the lab.
+
+Lab measurement is the best way to test the performance of features during
+development—before they've been released to users. It's also the best way to
+catch performance regressions before they happen.
+
+The following tools can be used to measure the Core Web Vitals in a lab
+environment:
+
+<table class="w-table-wrapper">
+  <thead>
+    <tr>
+      <th>&nbsp;</th>
+      <th>LCP</th>
+      <th>FID</th>
+      <th>CLS</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://developers.google.com/web/tools/chrome-devtools">
+        Chrome DevTools</a></td>
+      <td>✔</td>
+      <td>✘ (use <a href="/tbt/">TBT</a> instead)</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <td><a href="https://developers.google.com/web/tools/lighthouse">
+        Lighthouse</a></td>
+      <td>✔ (v6)</td>
```

Comment:
I suggest moving the `(v6)` to be on the td. `Lighthouse (v6)`. It applies to everything here.

Reply:
Yeah, I like that. Will update.

Reply:
Done.